sdktests/testapi_sdk_data.go

```diff
@@ -95,7 +100,9 @@ func NewSDKDataSourceWithoutEndpoint(t *ldtest.T, data mockld.SDKData, options .
 		d.streamingService = mockld.NewStreamingService(data, sdkKind, t.DebugLogger())
 	}
 
-	t.Debug("setting SDK data to: %s", string(data.Serialize()))
+	jsonStr, _ := json.Marshal(data)
```

Comment:
could check the error, then use `t` to assert it is nil

Reply:
Updated!